Week3/Solution/week3_solution.txt

```diff
@@ -0,0 +1,261 @@
+Write a python script to perform CRUD operation on following table of "ESM.db" database.
+
+Employee ( eid, ename, dept, basic, branch). eid must be primary key.
+
+Department : Account, Inventory, IT, HR.
+
+Peform Following Operation.
+
+1. Create table
+
+->
+from sqlite3 import *

```

Comment:
we write only import sqlite3  and access all functions from sqlite3. is that any specific reason to use 'from sqlite3 import *'?

Reply:
yes from this way we don't need to write sqlite3.connect()
we can directly used connect() by using from sqlite3 import *